floodviz/static/js/hydrograph.js

```diff
@@ -1,4 +1,3 @@
-
 // Set the dimensions of the canvas / graph
```

Comment:
Looks like you are using spaces rather than tabs for indentation. Please change to use tabs.